evm_loader/cli/src/config.rs

```diff
@@ -28,9 +28,7 @@ pub fn create(options: &ArgMatches) -> Result<Config, NeonError> {
             .unwrap_or(&solana_cli_config.json_rpc_url),
     );
 
-    let evm_loader = if let Some(value) = pubkey_of(options, "evm_loader") {
-        value
-    } else {
+    let Some(evm_loader) = pubkey_of(options, "evm_loader") else {
```

Comment:
Nice `let ... else` usage, though personally I prefer something like that:
```rust
let evm_loader = pubkey_of(options, "evm_loader").ok_or(NeonError::EvmLoaderNotSpecified)?;
```

Reply:
It was `clippy`'s idea, but I agree your variant is better. I will update it.